src/js/claims-status/components/ClaimsListItem.jsx

```diff
@@ -18,17 +18,17 @@ export default function ClaimsListItem({ claim }) {
         <p className="status"><span className="claim-item-label">Status:</span> {listPhase(claim.attributes.phase)}</p>
         <div className="communications">
           {inProgress && claim.attributes.developmentLetterSent
-            ? <p><i className="fa fa-envelope"></i>We sent you a development letter</p>
+            ? <p className="claim-list-item-text"><i className="fa fa-envelope claim-list-item-icon"></i>We sent you a development letter</p>
```

Comment:
I wonder if these should even be `p` elements? This is a list of items, so might be worth just changing the markup to a list

Reply:
Good point, change made.